index.js

```diff
@@ -31,18 +47,6 @@ module.exports.pitch = function (request) {
 		"// add the styles to the DOM",
 		"var update = require(" + loaderUtils.stringifyRequest(this, "!" + path.join(__dirname, "lib", "addStyles.js")) + ")(content, options);",
 		"if(content.locals) module.exports = content.locals;",
-		"// Hot Module Replacement",
-		"if(module.hot) {",
-		"	// When the styles change, update the <style> tags",
-		"	if(!content.locals) {",
-		"		module.hot.accept(" + loaderUtils.stringifyRequest(this, "!!" + request) + ", function() {",
-		"			var newContent = require(" + loaderUtils.stringifyRequest(this, "!!" + request) + ");",
-		"			if(typeof newContent === 'string') newContent = [[module.id, newContent, '']];",
-		"			update(newContent);",
-		"		});",
-		"	}",
-		"	// When the module is disposed, remove the <style> tags",
-		"	module.hot.dispose(function() { update(); });",
-		"}"
+		options.hmr ? hmr : ""
```

Comment:
To avoid breaking changes set by default to `true`

Reply:
Ohh 😲  good catch. Yep it **must** be `true` by default

Reply:
This is already handled by the `options.json`'s `default` value

Reply:
Also my unit test covers the default case by not having any values for this option and it won't break because the ajv handles the `default` value from `options.json` correctly.

Reply:
`options.hmr = options.hmr || true` (L19+) it shouldn't be mandatory to set this in `webpack.config.js` and it would be a breaking change aswell then. And please remove the `default: true` in case it causes a `ValidationError` for 

```js
{
   loader: 'style-loader',
   options: {}
},
{
   loader: 'style-loader',
   options: {
      other: option
   }
}
```

Reply:
Done and updated.